### PR TITLE
was added bootstrap

### PR DESCRIPTION
**Pull request by Hunter2065**
I added links for bootstrap in application.html.erb and also gem bootstrap in GemFile

### CODE REVIEW

**Review by montashevych — APPROVED** [review]
Resolve conflict. Other all fine.

**Comment by andrjuha01** [comment]
convert your css files to scss.
Also attach screenshot with rendered UI that shows that bootstrap is working fine

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,9 +8,11 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
```
Comment (andrjuha01) [inline app/views/layouts/application.html.erb:11]: shouldn't it be handled via gem?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,9 +8,11 @@
...
   </head>
 
   <body>
     <%= yield %>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
```
Comment (andrjuha01) [inline app/views/layouts/application.html.erb:16]: same

**Pull request closed without merging**